### DISCUSSION TITLE
How to generate functions/methods without named parameters

### DISCUSSION THREAD

**Discussion by Jhonfunk** [fzyzcjy/flutter_rust_bridge#1982] [Q&A] [answered]
You know, it's kind of time consuming to write the parameter names each time.

**Comment by fzyzcjy** [comment]
This is not supported yet (since nobody requested it before), but I will work on it in the next batch, alternatively feel free to PR!

**Comment by fzyzcjy** [comment, marked as answer]
Implemented

**Reply by Jhonfunk** [reply]
#1988 
Excellent! Now I don't have to write the parameter name every time. Thanks.

**Reply by fzyzcjy** [reply]
You are welcome!
